storm-server/src/main/java/org/apache/storm/daemon/metrics/reporters/ConsolePreparableReporter.java

```diff
@@ -46,13 +50,14 @@ public void prepare(MetricRegistry metricsRegistry, Map<String, Object> topoConf
             builder.convertDurationsTo(durationUnit);
         }
         reporter = builder.build();
+        reportingIntervalPeriod = ObjectReader.getInt(topoConf.get(DaemonConfig.STORM_DAEMON_METRICS_REPORTER_INTERVAL_SECS), 10);
```

Comment:
I'd like to see topoConf renamed here and in the other preparable reporters. The name is misleading, it isn't the topology config, but the daemon config. Renaming to e.g. "daemonConf" would make this clearer.

Reply:
Agreed